ruma-api-macros/src/api.rs

```diff
@@ -185,16 +188,18 @@ impl ToTokens for Api {
             TokenStream::new()
         };
 
-        let typed_response_body_decl = if self.response.has_body_fields()
-            || self.response.newtype_body_field().is_some()
-        {
-            let deserialized = response::try_deserialize(
-                quote! { ruma_api::exports::serde_json::from_slice(response.body().as_slice()) },
-            );
-            quote! { let response_body: ResponseBody = #deserialized; }
-        } else {
-            TokenStream::new()
-        };
+        let typed_response_body_decl =
+            if self.response.has_body_fields() || self.response.newtype_body_field().is_some() {
+                quote! {
+                    let response_body: ResponseBody =
+                        ::ruma_api::try_deserialize!(
+                            response,
+                            ruma_api::exports::serde_json::from_slice(response.body().as_slice()),
+                        );
```

Comment:
I think rustfmt would format this as

```suggestion
                    let response_body: ResponseBody = ::ruma_api::try_deserialize!(
                        response,
                        ruma_api::exports::serde_json::from_slice(response.body().as_slice()),
                    );
```

(you can check how rustfmt formats sth. by dropping `quote!` and replacing any `#` with `_` or any other characters valid in identifiers)